proto/sound_sensor.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+import "types/info.proto";
+
+// Trait for devices that measure sound/noise levels.
+service SoundSensorApi {
+  rpc GetSoundLevel(GetSoundLevelRequest) returns (SoundLevel) {}
+  rpc PullSoundLevel(PullSoundLevelRequest) returns (stream PullSoundLevelResponse) {}
+}
+
+service SoundSensorInfo {
+  rpc DescribeSoundLevel(DescribeSoundLevelRequest) returns (SoundLevelSupport) {}
+}
+
+// SoundLevel represents a sound/noise level.
+message SoundLevel {
+  // The measured sound/noise level, usually in dBA or dB. Use Info service to check.
+  float sound_level = 1;
```

Comment:
Best to make this `optional` I'd think, as 0 is a reasonable value in dB and we may need a way to indicate 'value unknown' as we do for other traits.